browser-support-molecule.css

```diff
@@ -608,6 +622,138 @@ body {
     height: 100%;
 }
 
+.version--selected::after {
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid;
+    content: '';
+    height: 0;
+    left: 24px;
+    position: absolute;
+    top: 63px;
+    width: 0;
+}
+
+.version--selected.not-supported::after {
+    border-bottom-color: #FF5A58;
+}
+
+.version--selected.partial-supported::after {
+    border-bottom-color: #F2AA52;
+}
+
+.version--selected.fully-supported::after {
+    border-bottom-color: #61D069;
+}
+
+.info-container {
+    display: none;
+    margin-top: 25px;
+    margin-bottom: 15px;
+}
+
+.info-container--show {
+    display: block;
+}
+
+.version-info {
+    border-radius: 2px;
+    color: #FFFFFF;
+    display: none;
+    padding: 25px 30px;
+}
+
+.version-info--selected {
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: row;
+    -ms-flex-direction: row;
+    flex-direction: row;
+}
+
+.version-info__section {
+    font-size: 18px;
+    font-weight: 400;
+}
+
+.version-info__section .title {
+    font-size: 20px;
+    font-weight: 500;
+    margin-bottom: 15px;
+}
+
+.version-info__section .note {
+    font-size: 16px;
+    font-weight: 300;
+    margin-top: auto;
+}
+
+.version-info__use {
+    flex-basis: 200px;
+    margin-right: 20px;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: column;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-align-items: flex-start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+}
+
+.usage-list {
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: column;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-align-items: flex-start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+    width: 100%;
+}
+
+.usage-list__item {
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: row;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-justify-content: space-between;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.usage-list__item__label {
+    font-size: 18px;
+    font-weight: 400;
+}
+.usage-list__item__data {
+    font-size: 18px;
+    font-weight: 500;
+}
+
+.version-info__support {
+    flex-basis: 300px;
+    margin: 0 20px;
+}
+
+.numbered-list__item {
+
+}
+
+.version-info__close {
```

Comment:
This should be absolutely positioned to the enclosing div, so that it follows the enclosing div as the enclosing div resizes.

```css
.version-info__close {
    font-size: 40px;
    font-weight: 300;
    cursor: pointer;
    position: absolute;
    top: 23px;
    right: 20px;
}
```